testing/upload_validation.ts

```diff
@@ -1611,6 +1612,15 @@ ${endpointKey ? 'endpointKey is set' : `requiresEndpointUrl is ${requiresEndpoin
           );
         };
 
+        const validateItemVersionProperty = () => {
+          return validateProperty(
+            'itemVersionProperty',
+            itemStatePropertySchema => itemStatePropertySchema.type === ValueType.String 
```

Comment:
While not public-facing, perhaps remove references to `itemState` if we are no longer using that term.